demoIsoObject.cpp

```diff
@@ -23,6 +23,7 @@ static po::variables_map parseArguments(
 	desc.add_options()
 		("help,h", "print this message")
 		("listen-ip,i", po::value<std::string>()->default_value("0.0.0.0"), "The IP address that the isoObject will listen on.")
+        ("behaviour,b", po::value<std::string>()->default_value("default"), "The behaviour of the isoObject. Options are 'default' and 'circle'")
```

Comment:
Maybe this can be called "follow-trajectory" or similar? instead of default, might be more descriptive.